args.c

```diff
@@ -94,8 +115,39 @@ void parse_args(int argc, char* argv[])
 			FTLfiles.log = "pihole-FTL.log";
 			FTLfiles.db = "pihole-FTL.db";
 			files.log = "pihole.log";
+			ok = true;
+		}
+
+		// List of implemented arguments
+		if(strcmp(argv[i], "-h") == 0 ||
+		   strcmp(argv[i], "help") == 0)
+		{
+			printf("pihole-FTL - The Pi-hole FTL engine\n\n");
+			printf("Usage:    sudo service pihole-FTL <action>\n");
+			printf("where '<action>' is one of start / stop / restart\n\n");
+			printf("Available arguments:\n");
+			printf("\td,  debug         More verbose logging,\n");
+			printf("\t                  don't go into daemon mode\n");
+			printf("\t    test          Don't start pihole-FTL but\n");
+			printf("\t                  instead quit immediately\n");
+			printf("\t-v, version       Return version\n");
+			printf("\t-t, tag           Return git tag\n");
+			printf("\t-b, branch        Return git branch\n");
+			printf("\t    running       Test if another pihole-FTL\n");
+			printf("\t                  process is running and exit\n");
+			printf("\t                  even if not (instead of\n");
+			printf("\t                  starting a new one)\n");
+			printf("\t-f, no-daemon     Don't go into daemon mode\n");
+			printf("\t-h, help          Display this help and exit\n");
+			printf("\n\nOnline help: https://github.com/pi-hole/FTL\n");
+			exit(EXIT_SUCCESS);
 		}
 
-		// Other arguments are ignored
+		// Complain if invalid options have been found
+		if(!ok)
+		{
+			printf("pihole-FTL: invalid option -- '%s'\nTry '%s --help' for more information\n", argv[i], argv[0]);
```

Comment:
Note that this will use `argv[0]` for the `help` command suggestion, i.e. if it is compiled locally in some subfolder and started via `some/folder/pihole-FTL abc` it will also know that and return:
```
$ some/folder/pihole-FTL abc
pihole-FTL: invalid option -- 'abc'
Try 'some/folder/pihole-FTL help' for more information
```